### PR TITLE
LIT 6917 to 20

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Hagiography of John of Wifat

### CODE REVIEW

File: new/LIT6919MiraclesYohWif.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <revisionDesc>
+            <change who="DN" when="2024-01-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:P129_is_about" active="LIT6919MiraclesYohWif" passive="PRS10419Yohanne"/>
+                </listRelation>
```
Comment (eu-genia) [inline new/LIT6919MiraclesYohWif.xml:54]: formsPartOf?

File: new/LIT6919MiraclesYohWif.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6919MiraclesYohWif" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracles of Yohannes of Wifat</title>
```
Comment (eu-genia) [inline new/LIT6919MiraclesYohWif.xml:8]: `xml:lang="en"` if you keep the English only

File: new/LIT6918ActsYohWif.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6918ActsYohWif" passive="LIT6917DossieYW"/>
```
Comment (eu-genia) [inline new/LIT6918ActsYohWif.xml:55]: isAbout?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6918ActsYohWif.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6918ActsYohWif.xml:55]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**